indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -38,12 +38,34 @@ PaidItemLocked.propTypes = {
   management: PropTypes.bool.isRequired,
 };
 
+function PurgedItemLocked({isUpdateMode}) {
+  const purgedMsg = (
+    <Translate>
+      This field is locked due to an expired retention period. The associated registration data have
+      been deleted.
+    </Translate>
+  );
+  const msg = !isUpdateMode ? (
+    purgedMsg
+  ) : (
+    <>
+      {purgedMsg} <Translate>The associated registration data have been deleted.</Translate>
```

Comment:
I think data is usually used in a singular way, ie *has been deleted*.

Also , this text seems to be repeated here since pretty much the same string is already part of `purgedMsg`